src/base/utils/misc.cpp

```diff
@@ -65,6 +65,8 @@
 #if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC)) && defined(QT_DBUS_LIB)
```

Comment:
Isn't `#ifdef QT_DBUS_LIB` enough?

Reply:
`QT_DBUS_LIB` could theoretically be true on macOS and Windows too...

Reply:
>QT_DBUS_LIB could theoretically be true on macOS and Windows too...

Yes, but only if we use DBus module (i.e. `QT += dbus` in config). We use it only on Unix so it's defined only on Unix.
In other words, if it defined on Windows here (or MacOS) then we do use it!

Reply:
You are correct.
But in this file the dbus interfaces we're using are only available on Linux. https://github.com/qbittorrent/qBittorrent/blob/master/src/base/utils/misc.cpp#L170
So I don't like lessening the ifdef in this case.